Source/Events.Store.MongoDB/Streams/Filters/UnsupportedFilterTypeEnumValue.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Dolittle.Runtime.Events.Store.MongoDB.Streams.Filters
+{
+    /// <summary>
+    /// Exception that gets thrown when a <see cref="AbstractFilterDefinition"/> document has an unsupported Type field.
+    /// </summary>
+    public class UnsupportedFilterTypeEnumValue : Exception
```

Comment:
```suggestion
    public class UnsupportedFilterTypeEnumValue : EventStoreConsistencyError
```

Reply:
Fixed and added a `null` as the given innerException to the constructor.